components/MDX.js

```diff
@@ -0,0 +1,153 @@
+          </div>
+        </div>
+      </div>
+    ),
+    th: props => <th {...props} className={`${props.className || ''} px-6 py-3 border-b border-gray-200 bg-gray-100 text-left text-xs leading-4 font-medium font-body text-gray-900 uppercase tracking-wider`} />,
+    tr: props => <tr {...props} className={`${props.className || ''} bg-white`} />,
+    td: props => <td {...props} className={`${props.className || ''} px-6 py-4 border-b border-gray-200 text-sm leading-5 text-gray-700 tracking-tight`} />,
+    pre: props => <div {...props} className={`${props.className || ''} my-8`} />,
+    inlineCode: props => <code {...props} className={`${props.className || ''} px-1 py-0.5 bg-gray-200 text-gray-800 rounded font-mono text-sm`} />,
+    code: CodeComponent,
+    hr: props => <hr {...props} className={`${props.className || ''} my-8`} />,
+    CodeBlock,
+    ChapterSuggestions,
+    YouTube,
+    DocsButton,
+    Remember,
+    Warning,
+    Sponsors,
+    Caption,
+    Link,
+    Row,
+    Column,
+    Figure,
+    DocsCards,
+    GeneratorInstallation,
+    NewsletterSubscribe,
+    TwitterTimelineEmbed,
+    TwitterShareButton,
+    TwitterFollowButton,
+    TwitterHashtagButton,
+    TwitterMentionButton,
+    TwitterTweetEmbed,
+    TwitterMomentShare,
+    TwitterDMButton,
+    TwitterVideoEmbed,
+    TwitterOnAirButton
+  }
+}
+
+function CodeComponent({ children, className = '', metastring = '', ...rest }) {
+  let caption
+  const meta = metastring.split(/([\w]+=[\w\d\s\-_:><.]+)/) || []
+  meta.forEach(str => {
+    const params = new URLSearchParams(str)
+    caption = params.get('caption') || ''
+    if (caption.startsWith("'") && caption.endsWith("'")) caption = caption.substring(1, caption.length - 1)
+  })
+  const maybeLanguage = className.match(/language\-([\w\d\-_]+)/)
+  const language = maybeLanguage && maybeLanguage.length >= 2 ? maybeLanguage[1] : undefined;
+
+  if (language === 'mermaid') {
+    return (
+      <MermaidDiagram graph={children} />
+    )
+  }
+
+  return (
+    <CodeBlock {...rest} caption={caption} className={`${className || ''} rounded`} language={language} showLineNumbers={children.split('\n').length > 2}>
+      {children}
+    </CodeBlock>
+  )
+}
+
+let currentId = 0;
+const uuid = () => `mermaid-${(currentId++).toString()}`;
+
+function MermaidDiagram({ graph }) {
+  const [svg, setSvg] = useState(null);
+
+  useLayoutEffect(() => {
+    if (!graph) {
+      return;
+    }
+
+    try {
+      mermaid.mermaidAPI.render(uuid(), graph, renderedSvg => setSvg(renderedSvg));
+    } catch (e) {
+      setHtml(null)
+      console.error(e);
+    }
+  }, [graph]);
+
+  return svg ? <div dangerouslySetInnerHTML={{ __html: svg }} /> : null;
+}
+
+function initializeMermaid() {
+  if (mermaidInitialized) {
+    return;
+  }
+
+  mermaidInitialized = true;
+  mermaid.initialize({
+    startOnLoad: true,
+    theme: "default",
+    securityLevel: "loose",
+    themeCSS: ``,
```

Comment:
```suggestion
    themeCSS: ``,
    themeVariables: {
      primaryColor: "#EDFAFF",
      primaryBorderColor: "#47BCEE",
      secondaryColor: "#F4EFFC",
      secondaryBorderColor: "#875AE2",
      fontFamily: "Inter, sans-serif",
      fontSize: "18px",
      primaryTextColor: "#242929",
      tertiaryColor: "#F7F9FA",
      tertiaryBorderColor: "#BFC6C7"
    }
```

Here are some colors that I am suggesting we add instead of the current theme.